orchestrator/api/codegraph.py

```
@@ -72,15 +66,26 @@ class ProjectResponse(BaseModel):
     created_at: Optional[str]
 
 
-# Helper to get service instance
+# Helper functions
+def get_openai_key() -> str:
+    """Lazy load OpenAI API key from credentials or environment"""
+    try:
+        from services.credential_resolver import get_credential_resolver
+        resolver = get_credential_resolver()
+        return resolver.get_credential_field("development_openai", "api_key")
+    except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)